templates/ClientInstance.ts.hbs

```diff
@@ -6,8 +6,8 @@ import { OperationOptions } from "retry";
 // eslint-disable-next-line @typescript-eslint/no-namespace
 export namespace {{name.upperCamelCase}} {
   {{#each dataTypes}}
-      {{#if (isTypeDefinition .)}}
-      export type {{getValue name}} = {{> dtoPartial typeDto=.}}
+      {{#if (or (isTypeDefinition this) (isArrayType this))}}
+export type {{getValue name}} = {{> dtoPartial typeDto=.}}
```

Comment:
whys is "dot" replaced with "this"?should we use "this" at line #10 as well to be consitent

Reply:
I think this is just the nicer way to doing it. But the effect is the same. I'll change it on line 10 too.